doc/sphinx/practical-tools/coqide.rst

```diff
@@ -86,16 +87,24 @@ Running Coq scripts
 Operations for running the script are available in the *Navigation* menu,
 from the toolbar and from the keyboard.  These include:
 
-- Forward (`Ctrl-Down`) to run one command or tactic
-- Backward (`Ctrl-Up`) undo one command or tactic
-- Run to cursor (`Ctrl-Right`) to run commands up to the cursor
-- Run to end (`Ctrl-End`) to run commands to the end of the buffer
-- Reset Coq (`Ctrl-Home`) to restart the Coq process
+- Forward (`Alt-Down`) to run one command or tactic
+- Backward (`Alt-Up`) undo one command or tactic
+- Run to cursor (`Alt-Right`) to run commands up to the cursor
+- Run to end (`Alt-End`) to run commands to the end of the buffer
+- Reset Coq (`Alt-Home`) to restart the Coq process
 - Interrupt to stop processing commands after the current command completes.
   (Note: on Windows but not on WSL, Interrupt doesn't work if you start CoqIDE
   as a background process, e.g. `coqide &` in bash.  See Coq issue
   `#16142 <https://github.com/coq/coq/pull/16142>`_).
 
+On macOS, use `Cmd-Ctrl` instead of `Alt` for these operations.
+
+Note about the change of modifier: the modifier for Navigation was previously
+`Ctrl` (except on macOS). It was changed so that the hotkeys for moving the
+cursor in the editor (`Ctrl-Right`, `Ctrl-Home`, `Ctrl-End`) are supported by
+default. To restore the previous behavior, go to *Edit/Preferences/Shortcuts*
+and change the modifier for Navigation.
+
```

Comment:
I think this is covered by the change log, so it's no longer needed.  (This text quickly becomes out of date after a couple releases.)